Cleanup compose app tutorial (#555)
foundriesio/docs: source/tutorials/compose-app/compose-app-testing-applications.rst

    device:~$ docker logs -f shellhttpd-mqtt_httpd-mqtt_1

In a second terminal, test your apps using ``curl``.
Use any external device connected to the same network such as your computer.

.. prompt:: bash host:~$, auto

    host:~$ #Example curl 192.168.15.11:8082
    host:~$ curl <device IP>:8082

::

     Number of Access = 1

At the same time, in the first terminal connected to your device, 
the ``shellhttpd-mqtt_httpd-mqtt_1`` logs should show:

::

     Number of Access = 1
     ----------------------

Now, test ``flask-mqtt-nginx_flask-mqtt_1``.
First, follow the container log on your device with the command:

.. prompt:: bash device:~$, auto

    device:~$ docker logs -f flask-mqtt-nginx_flask-mqtt_1

In a second terminal, test your apps using ``curl``.

> Use any external device connected to the same network such as your computer.

Reviewer: ```suggestion
Use any external device connected to the same network, such as your computer.
```

.. prompt:: bash host:~$, auto

    host:~$ #Example curl 192.168.15.11:80
    host:~$ curl <device IP>:80

::

     Number of Access on shellhttpd Container 1

In the first terminal connected to your device, 
the ``flask-mqtt-nginx_flask-mqtt_1`` logs should show:

::

     172.20.0.3 - - [] "GET / HTTP/1.0" 200 -

Now, as you access the ``shellhttpd-mqtt_httpd-mqtt_1`` app, 

> ``flask-mqtt-nginx_flask-mqtt_1`` will track how many accesses there has been.

Reviewer: ```suggestion
``flask-mqtt-nginx_flask-mqtt_1`` will track how many accesses there have been.
```

> The total count willbe displayed when you check the port 80.

Reviewer: ```suggestion
The total count will be displayed when you check port 80.
```
